.github/workflows/template-janitor.yml

```
@@ -41,17 +46,17 @@ jobs:
           sed -i "s/myproject/${{ github.event.repository.name }}/gi" CMakeLists.txt configured_files/config.hpp.in
 
           # Update URL placeholders for project
-          sed -i "s/%%myurl%%/${{ github.event.repositoryUrl }}/gi" CMakeLists.txt
+          sed -i "s|%%myurl%%|${{ github.event.repositoryUrl }}|gi" CMakeLists.txt
```

```suggestion
          sed -i "s|%%myurl%%|${{ env.NEW_URL }}|gi" CMakeLists.txt
```
Somehow this ended up empty in the last run